macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionTunnelController.swift

```diff
@@ -547,49 +551,18 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
         controllerErrorStore.lastErrorMessage = nil
 
         do {
-#if NETP_SYSTEM_EXTENSION
```

Comment:
Just moved this all into a method that supports retrying if the configuration is invalid.